policy/modules/kernel/corenetwork.te.in

```diff
@@ -32,6 +32,10 @@ dev_node(ppp_device_t)
 type tun_tap_device_t;
 dev_node(tun_tap_device_t)
 
+optional_policy(``
+	container_mountpoint(tun_tap_device_t)
+'')
```

Comment:
Extra quotes.

Reply:
This was intentional to avoid a lint error since the corenetwork module is an m4 template: https://github.com/0xC0ncord/refpolicy/runs/4549749915

Other solutions would be to write an m4 define in `corenetwork.te.m4` for `optionalpolicy()` or bring back the previous interface to allow `mounton` access on `tun_tap_device_t`.

Reply:
Ok. Please add a comment to explain this.